lib/matplotlib/backend_bases.py

```diff
@@ -293,13 +293,14 @@ def draw_path_collection(self, gc, master_transform, paths, all_transforms,
         """
         path_ids = []
         for path, transform in self._iter_collection_raw_paths(
-                master_transform, paths, all_transforms):
+                master_transform, paths, all_transforms, offsets):
```

Comment:
I honestly don't know if there is a problem or not. Only that typically elsewhere in mpl, we have the transforms for the offset independent of the transform for other aspects of an artist, I think. So, to see the offsets passed into a function, but not the transform for the offset, I start wondering if this might be subtly broken?